CHANGELOG.md

```diff
@@ -4,6 +4,64 @@ All notable changes to this project will be documented in this file.
+- [#589](https://github.com/Automattic/VIP-Coding-Standards/pull/589): WPQueryParams: inherit from abstract class and flag for 'exclude' parameter.
+- [#592](https://github.com/Automattic/VIP-Coding-Standards/pull/592): DynamicCalls: various improvements.
+- [#595](https://github.com/Automattic/VIP-Coding-Standards/pull/595): Underscorejs: various improvements.
+- [#620](https://github.com/Automattic/VIP-Coding-Standards/pull/620): Ruleset: silence UnusedVariable from VariableAnalysis to reduce noise.
+- [#624](https://github.com/Automattic/VIP-Coding-Standards/pull/624): ProperEscapingFunction: privatize $escaping_functions property along with various improvements.
+- [#635](https://github.com/Automattic/VIP-Coding-Standards/pull/635): Ruleset: remove WordPress.WP.GlobalVariablesOverride.Prohibited severity declaration.
+- [#638](https://github.com/Automattic/VIP-Coding-Standards/pull/638): IncludingFile: allow custom keywords to reduce false positives.
+- [#639](https://github.com/Automattic/VIP-Coding-Standards/pull/639): RestrictedFunctions: remove site_option group.
+- [#644](https://github.com/Automattic/VIP-Coding-Standards/pull/644): RestrictedFunctions: remove wp_cache_get_multi group.
+- [#645](https://github.com/Automattic/VIP-Coding-Standards/pull/645): Ruleset: silence WordPress.WP.AlternativeFunctions.file_system_read_readfile.
+- [#646](https://github.com/Automattic/VIP-Coding-Standards/pull/646): Ruleset: silence WordPress.WP.AlternativeFunctions.file_system_read_fclose.
+- [#647](https://github.com/Automattic/VIP-Coding-Standards/pull/647): RestrictedFunctions: remove get_super_admins group.
+- [#649](https://github.com/Automattic/VIP-Coding-Standards/pull/649): RestrictedFunctions: downgrade switch_to_blog() to warning and change messaging.
+- [#652](https://github.com/Automattic/VIP-Coding-Standards/pull/652): RestrictedFunctions/RestrictedVariables: remove usermeta related errors.
+
+## Fixed
+- [#444](https://github.com/Automattic/VIP-Coding-Standards/pull/444): ConstantString: account for namespace prefixing.
+- [#581](https://github.com/Automattic/VIP-Coding-Standards/pull/581): AlwaysReturnInFilter: fix runtime failure on abstract methods.
+- [#586](https://github.com/Automattic/VIP-Coding-Standards/pull/586): IncludingNonPHPFile: various bug fixes.
+- [#587](https://github.com/Automattic/VIP-Coding-Standards/pull/587): LowExpiryCacheTime: allow arithmetic operators, simple floats, numerical strings, zeroes and parantheses in calculations, and FQN time constants.
+- [#592](https://github.com/Automattic/VIP-Coding-Standards/pull/592): DynamicCalls: ignore comments, allow double quotes and remove potential memory leak.
+- [#595](https://github.com/Automattic/VIP-Coding-Standards/pull/595): Underscorejs: no error shown during escaped variable.
+- [#654](https://github.com/Automattic/VIP-Coding-Standards/pull/654): LowExpiryCacheTime: improve handling of parse errors in evaluated code.
```

Comment:
This doesn't need an entry as it is a bug fix cleaning up after #587. I.e. the bug was never in a release.